src/sonic-pac/mab/mapping/include/mab_socket.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 Broadcom Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#ifndef MAB_SOCKET_H
+#define MAB_SOCKET_H
+
+typedef struct mab_pac_cmd_s
+{
+  char intf[128];
+  char cmd[128];
```

Comment:
Can you replace the hardcoded values with enums?

Reply:
Updated with enum